tests/health_checks/NDv4/run-health-checks.sh

```diff
@@ -7,12 +7,44 @@
 #Verify if an argument was passed to the script. Test if the argument is
 #a directory that contains all the expected health checks. If no arguments
 #are passed use the default directory.
+declare -a files=("check-hardware.sh" "dcgmi-check.sh" "ecc-errors.sh"\
+	   "test-bandwidth.sh" "test-ib.sh" "test-nccl.sh"\
+	   "test-MaxPayLoad.sh")
 if [ "$#" -eq 0 ]; then
-	TEST_DIR="/opt/azurehpc/test/health_checks/NDv4/"
+	pass=1
+	DIR=$(cd `dirname $0` && pwd)
+	for file in "${files[@]}"; do
+		if test -f "$DIR/$file"; then
+			true
+		else
+			pass=0
+		fi
+	done
+	#if tests are located in script dir use those
+	if [[ $pass -eq 1 ]]; then
+		TEST_DIR=$DIR
+	fi
+	if [[ $pass -eq 0 ]]; then
```

Comment:
I decided to remove the absolute path so we are no longer assuming anything about the directory structure where the users are running these codes. This way you can just pull the test scripts from github on any machine and run them, and it will work, even if that directory isn't there.

Reply:
great